### PR TITLE
Added mod alerted notice to auto-infractions

### LINKED ISSUE #2247: Modify bot's automute messages to indicate that moderators role has already been pinged
Currently, automutes by the bot ping the Moderators role in a mod-alerts channel, and also send a public message indicating the user has been muted:
image

Well-meaning bystanders then ping moderators to have them unmuted, which is usually redundant. (If the user wasn't already unmuted, then that means either a moderator decided to leave the mute, extend it, or, there is no moderator online to action the incident anyways.)

We would like to change the message to more clearly indicate that the moderators have already been alerted. Something to the effect of: "Moderators have already been alerted and will review."

### PR DESCRIPTION
**Pull request by ionite34**
Closes #2247 

## Summary
- Added notice that moderators have been alerted to all automated infractions.

## Implementation
[infraction/_scheduler.py](https://github.com/python-discord/bot/compare/automute-message?expand=1#diff-2ffa4e37dc82fe6fd91f8f63c1fe65bdb11f2619f5ed3fbb352f1573361998f2)
```diff
if reason:
-   end_msg = f" (reason: {textwrap.shorten(reason, width=1500, placeholder='...')})"
+   end_msg = (
+       f" (reason: {textwrap.shorten(reason, width=1500, placeholder='...')})"
+       " Moderators have been alerted for review"
+   )
```

## Demo
img

### CODE REVIEW

File: bot/exts/moderation/infraction/_scheduler.py [hunk 1]
```diff
@@ -189,7 +189,10 @@ async def apply_infraction(
                 f"Infraction #{id_} actor is bot; including the reason in the confirmation message."
             )
             if reason:
-                end_msg = f" (reason: {textwrap.shorten(reason, width=1500, placeholder='...')})"
+                end_msg = (
+                    f" (reason: {textwrap.shorten(reason, width=1500, placeholder='...')})"
```
Comment (wookie184) [inline bot/exts/moderation/infraction/_scheduler.py:193]:
```suggestion
                    f" (reason: {textwrap.shorten(reason, width=1500, placeholder='...')})."
```
Maybe?

Comment (minalike) [reply bot/exts/moderation/infraction/_scheduler.py:193]: Can we add 2 line breaks between the 2 lines?

Comment (ionite34) [reply bot/exts/moderation/infraction/_scheduler.py]: Added period, newlines and non-mentioned moderator role in e68868e

img

**Review by wookie184 — APPROVED** [review]
LGTM

**Comment by ionite34** [comment]
Current latest style as of [c9d5aac](https://github.com/python-discord/bot/pull/2251/commits/c9d5aac1d71bdbee672355e75ee8e655892c8b6c):
img

File: bot/exts/moderation/infraction/_scheduler.py [hunk 2]
```diff
@@ -189,7 +189,10 @@ async def apply_infraction(
                 f"Infraction #{id_} actor is bot; including the reason in the confirmation message."
             )
             if reason:
-                end_msg = f" (reason: {textwrap.shorten(reason, width=1500, placeholder='...')})"
+                end_msg = (
+                    f" (reason: {textwrap.shorten(reason, width=1500, placeholder='...')})."
+                    f"\n\nThe <@&{Roles.moderators}> have been alerted for review"
```
Comment (minalike) [inline bot/exts/moderation/infraction/_scheduler.py:194]:
```suggestion
                    f"\n\nThe <@&{Roles.moderators}> have been alerted for review."
```

**Review by minalike — APPROVED** [review]

**Pull request merged**